charts/airflow/files/pod_template.kubernetes-helm-yaml

```diff
@@ -38,14 +38,17 @@ spec:
   securityContext:
     {{- $podSecurityContext | nindent 4 }}
   {{- end }}
-  {{- if or ($extraPipPackages) (.Values.dags.gitSync.enabled) }}
+  {{- if or ($extraPipPackages) (.Values.dags.gitSync.enabled) (.Values.kubernetesPodTemplate.extraInitContainers) }}
   initContainers:
     {{- if $extraPipPackages }}
     {{- include "airflow.init_container.install_pip_packages" (dict "Release" .Release "Values" .Values "extraPipPackages" $extraPipPackages) | indent 4 }}
     {{- end }}
     {{- if .Values.dags.gitSync.enabled }}
     {{- include "airflow.container.git_sync" (dict "Release" .Release "Values" .Values "sync_one_time" "true") | indent 4 }}
     {{- end }}
+    {{- if .Values.kubernetesPodTemplate.extraInitContainers }}
```

Comment:
This is incorrect, the value is `airflow.kubernetesPodTemplate.extraInitContainers`, not `kubernetesPodTemplate.extraInitContainers`.